new_sem8/dz1.py

```diff
@@ -1,3 +1,24 @@
 # Напишите функцию, которая преобразует pickle файл хранящий список словарей в табличный csv файл
 # Для тестированию возьмите pickle версию файла из предыдущей задачи.
 # Функция должна извлекать ключи словаря для заголовков столбца из переданного файла.
+
+import pickle
+import csv
+
+# dict_list = [{'one': '111', 'two': '111'},
+#           {'one': '222', 'two': '222'},
+#           {'one': '333', 'two': '333'},]
+
+def pickle_to_csv(pickle_file: str, csv_file: str):
+    pickle_obj = pickle.loads(open(pickle_file, 'rb').read())
+    pickle_obj = pickle.dumps(pickle_obj)
+    pickle_obj = pickle.loads(pickle_obj)
+
+    with open('data.csv', 'w', newline='') as cf:
```

Comment:
Используйте параметр, который у вас определен в функции
```suggestion
    with open(csv_file, 'w', newline='') as cf:
```